test.js

```diff
@@ -96,6 +96,40 @@ ${padding}└───┘
 ${padding}    `);
 });
 
+test('float option (right) with margin', t => {
+	const padSize = Math.max(process.stdout.columns - 4 - 6, 0) - 1;
```

Comment:
Can you move `6` out into a more descriptive variable and use that variable there? Same with the below test.